jobs/aide/templates/bin/run-report.sh.erb

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -x
+
+JOB_DIR=/var/vcap/jobs/aide
+PACKAGE_DIR=/var/vcap/packages/aide
+
+$PACKAGE_DIR/bin/aide --check > report.txt
+
+# aide exits nonzero when it finds a modification, so wait until here to set -e
```

Comment:
nitpicks:  I'd rather `aide --check > file || true` instead of `set +e ... set -e`

(but this isn't a blocker)

Reply:
Tbh I prefer the cognitive ease of `set` vs `||` in bash.